.lagoon.yml

```
@@ -86,6 +86,12 @@ tasks:
             echo "Test users already exist. Skipping creation..."
           fi
         service: cli
+    - run:
+        name: Enable example content
+        command: |
+          set -e
+          drush en -y dpl_example_content
```

```suggestion
        command: drush en -y dpl_example_content
```

There is no need for `set -e` for single line commands. I understand if you want to keep it around for consistency though.